library/src/main/java/com/novoda/downloadmanager/lib/Request.java

```diff
@@ -246,6 +247,15 @@ public Request allowTarUpdates() {
     }
 
     /**
+     * When a ETag header is present, the application should check the integrity of the
+     * downloaded file, otherwise the current download won't be able to be resumed
```

Comment:
I don't think that is possible, or at least it is a violation of the spec so maybe we don't need to handle this? http://tools.ietf.org/html/rfc7232#section-2.3